### ISSUE TITLE
Cannot read property 'cwd' of undefined

### ISSUE THREAD

**Issue by Graham42** [jonschlinkert/copy#4]
My code fairly simple but maybe I'm missing something

``` js
var copy = require('copy');
var OUTPUT_DIR = '_output/';
copy.sync('src/*.js', OUTPUT_DIR );
```

I get this stacktrace

```
Error: Error: TypeError: Cannot read property 'cwd' of undefined
    at /home/graham/dev/<my_app>/node_modules/copy/index.js:55:13
    at Array.forEach (native)
    at Function.copySync [as sync] (/home/graham/dev/<my_app>/node_modules/copy/index.js:51:31)
    at Object.<anonymous> (/home/graham/dev/<my_app>/build.js:17:6)
    ...
```

**Comment by doowb** [comment]
There was a bug fix for this in #3 but wasn't published to npm. I published the fix as `0.1.3` so it should work after getting the latest.

**Comment by jonschlinkert** [comment]
sorry, yeah I think I got distracted with the refactor. thanks @doowb!
